DQMServices/Core/interface/DQMStore.h

```diff
@@ -792,11 +792,15 @@ namespace dqm {
 
       // Book MEs by lumi by default whenever possible.
       bool doSaveByLumi_;
-      std::vector<std::string> MEsToSave_;  //just if perLS is ON
```

Comment:
Thanks for your comments @perrotta 
Defintely this variable must be now public, so I am removing this commented line which I had forgotten

Reply:
It would have been better to add an accessor that returns a const reference to the `vector<string>` (which `IGetter` could use too instead of friendship), or declare `IBooker` as a friend to `DQMStore`. Now anyone can easily change `DQMStore::MEsToSave_`.

Reply:
Hi @makortel 
Thanks for your feedback. I have tried to implement your suggestiong but a pointer gave me problems with the range based loop suggested by @perrotta so I had to do a reference instead as in:

https://github.com/cms-sw/cmssw/commit/23623b7af9578ca702c2a6b27443964d9d80ec7b

otherwise I got:
no matching function for call to 'begin(const std::vector<std::__cxx11::basic_string<char> >* const&)
107 |         for (auto& thepath : MEs) {
      |                              ^~~

Reply:
@perrotta @makortel  would this commit above be ok for you? Thanks

Reply:
@jfernan2 I'd let @makortel comment on the technical implementation.
In any case, if the range-based for loop was causing problems to you because of lacking of begin/end methods in the collection, feel free to revert it back.